src/panels/lovelace/cards/hui-sensor-graph-card.js

```diff
@@ -0,0 +1,278 @@
+import { LitElement, html, svg } from '@polymer/lit-element';
+
+import '../../../components/ha-card.js';
+import '../../../components/ha-icon.js';
+
+import computeStateName from '../../../common/entity/compute_state_name.js';
+import stateIcon from '../../../common/entity/state_icon.js';
+
+import EventsMixin from '../../../mixins/events-mixin.js';
+
+class HuiSensorGraphCard extends EventsMixin(LitElement) {
+  set hass(hass) {
+    this._hass = hass;
+    const entity = hass.states[this.config.entity];
+    if (entity && this.entity !== entity) {
+      this.entity = entity;
+      this._getHistory();
+    }
+  }
+
+  static get properties() {
+    return {
+      _hass: { },
+      config: { },
+      entity: { },
+      line: String
+    };
+  }
+
+  setConfig(config) {
+    if (!config.entity || config.entity.split('.')[0] !== 'sensor') {
+      throw new Error('Specify an entity from within the sensor domain.');
+    }
+
+    const cardConfig = Object.assign({
+      icon: false,
+      hours_to_show: 24,
+      accuracy: 10,
+      height: 100,
+      line_width: 5,
+      line_color: 'var(--accent-color)'
+    }, config);
+    cardConfig.hours_to_show = Number(cardConfig.hours_to_show);
+    cardConfig.accuracy = Number(cardConfig.accuracy);
+    cardConfig.height = Number(cardConfig.height);
+    cardConfig.line_width = Number(cardConfig.line_width);
+
+    this.config = cardConfig;
+  }
+
+  shouldUpdate(changedProps) {
+    const change = (
+      changedProps.has('entity')
+      || changedProps.has('line')
+    );
+    return change;
+  }
+
+  render({ config, entity, line } = this) {
+    return html`
+      ${this._style()}
+      <ha-card @click='${e => this._handleClick(e)}'>
```

Comment:
Defining an arrow function is no longer needed in Lit Element 0.6.2. It will now set the event context correctly and you can do `@click='${this._handleClick}'`